regression/contracts/assigns_enforce_15/main.c

```diff
@@ -0,0 +1,20 @@
+int foo(int *x) __CPROVER_assigns(*x)
+  __CPROVER_ensures(__CPROVER_return_value == *x + 5)
+{
+  int z = 5;
+  int y = bar(&z);
+  return *x + 5;
+}
+
+int bar(int *a) __CPROVER_ensures(__CPROVER_return_value >= 5)
+{
+  *a = *a + 2;
+  return *a;
+}
+
+int main()
```

Comment:
Can we add another function that doesn't have assigns clause, but modifies a global variable?
We can check for the appropriate assertion in the `test.desc` file instead of creating a separate regression test just for this case.

Reply:
Implemented as suggested.